squidb/src/com/yahoo/squidb/utility/VersionCode.java

```diff
@@ -18,6 +18,13 @@
     public static final VersionCode V3_7_4 = new VersionCode(3, 7, 4, 0); // Default on API 14, default minimum
```

Comment:
I could go either way between having a collection of salient versions here or just making one-offs where needed. Since there only seems to be one, maybe the one-off is the better approach.